packages/flutter_tools/lib/src/commands/build_web.dart

```diff
@@ -15,8 +15,10 @@ import 'build.dart';
 
 class BuildWebCommand extends BuildSubCommand {
   BuildWebCommand({
+    required super.logger,
+    required FileSystem fileSystem,
     required bool verboseHelp,
```

Comment:
as a further cleanup, I think this can be `required super.verboseHelp,` and then we don't have to explicitly call `super(verboseHelp: verboseHelp)`